applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -266,6 +266,33 @@ source "subsys/logging/Kconfig.template.log_config"
 
 endmenu # Log levels
 
+#----------------------------------------------------------------------------#
+config AUDIO_MIN_PRES_DLY_US
+	int "The minimum presentation delay"
+	default 4000
+	help
+	  The minimum presentation delay determined by the audio system processing
+	  and the minimum buffering.
+
+config AUDIO_MAX_PRES_DLY_US
```

Comment:
Why are there duplicate configs?

Reply:
There are not duplicates as they have different meanings.